### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by Baxterthedog1** [qwqdanchun/DcRat#51]

**Comment by Dazmed707** [comment]
Create a malware .exe, configure ip and port and execute in your pc the victim.

**Comment by 9993rills** [comment]
have maybe a silly question. dc rat works even if port is CLOSED??

**Comment by qwqdanchun** [comment]
no



---Original---
From: ***@***.***&gt;
Date: Mon, Aug 30, 2021 15:30 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] how to use (#51)




 
have maybe a silly question. dc rat works even if port is CLOSED??

**Comment by 9993rills** [comment]
tried with a configured ip and opened port 433 no report from target but with my local ip with closed port on same target i get report (target on diff ip though)... kinda confusing... ok

**Comment by qwqdanchun** [comment]
maybe you mean 127.0.0.1:443 ?
this don't need open port



---Original---
From: ***@***.***&gt;
Date: Mon, Aug 30, 2021 15:37 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] how to use (#51)




 
tried with a configured ip and opened port 433 no report from target but with my local ip with closed port on same target i get report (target on diff ip though)... kinda confusing... ok

**Comment by 9993rills** [comment]
no, i already removed that long ago.... my real local ip 192.168.xxx.xxx

**Comment by qwqdanchun** [comment]
if i don't misunderstand,the ip you use is just you are using 127.0.0.1



---Original---
From: ***@***.***&gt;
Date: Mon, Aug 30, 2021 15:41 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] how to use (#51)




 
no, i already removed that long ago.... my real local ip 192.168.xxx.xxx

**Comment by 9993rills** [comment]
i must be getting it wrong somwhr then.... when i set ip and port it goes into builder's tab right?? i set ip 192.168.xxx.xxx from my local host not 172.0.0.1 already removed that totally, and set port on 443 already cleared default port... now it connect on my pc and even when i sent backdoor to target on different ip it also connect although port 443 is closed on my pc

ok i thought this is good, and then i opened a port 443, good port opened ip changed and tried on same target no connection... thats my surprise anyway!!!

**Comment by qwqdanchun** [comment]
can't really understand what you mean ,but usually you should not use ip like 192.168.*.* on your rat



---Original---
From: ***@***.***&gt;
Date: Mon, Aug 30, 2021 15:50 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] how to use (#51)




 
i must be getting it wrong somwhr then.... when i set ip and port it goes into builder's tab right?? i set ip 192.168.xxx.xxx from my local host not 172.0.0.1 already removed that totally, and set port on 443 already cleared default port... now it connect on my pc and even when i sent backdoor to target on different ip it also connect although port 443 is closed on my pc
 
ok i thought this is good, and then i opened a port 443, good port opened ip changed and tried on same target no connection... thats my surprise anyway!!!

**Comment by 9993rills** [comment]
so what kind of ip shld i use, help pls?!!

**Comment by 9993rills** [comment]
dont knw how to use pastebin, and no-ip i'm really not getting anywhere with that too.... maybe somthing i should know here, any help pls??!!

**Comment by qwqdanchun** [comment]
just buy a server and use it to run the rat

**Comment by 9993rills** [comment]
OH ok i have windows server 2012 r2 running in VM, still setting it up tho! is that okay enough? thanks!

**Comment by qwqdanchun** [comment]
i think you should learn about ip and network first😂


you need to have a ip on Internet,not your local ip like 192.168.*.*



---Original---
From: ***@***.***&gt;
Date: Mon, Aug 30, 2021 16:11 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] how to use (#51)




 
OH ok i have windows server 2012 r2 running in VM, still setting it up tho! is that okay enough? thanks!

**Comment by 9993rills** [comment]
Yah! really worth d laff, Lol!! i alrdy tried ip on internet never got connections too, just like you cant understand how i'm able to make a connection with closed port same as how i can not understand why internet ip would not work at my end in the first place.... THANKS FOR YOUR LECTURE THO! nice RAT too...

**Comment by Dazmed707** [comment]
If you want write me to telegram I will help you to do it on your computer without buying a server
@dazmed707

**Comment by protataetuk** [comment]
use ngrok and pastebin

**Comment by 9993rills** [comment]
Thanks

**Comment by 9993rills** [comment]
> use ngrok and pastebin

Thanks

**Comment by PhillipDScottJr** [comment]
Use NGROk to forward your ports..you may not be getting a connection because of your ports are not forwarded.  When you are attacking a computer on your network then use 127.0.0.1 or if your on windows call ipconfig an detect your INTERNAL zip address..if you are attacking a machine outside of your network like let's say your girlfriends house then you would use your public option address..your public ip can be found on ip-score.net.

**Comment by 9993rills** [comment]
> Use NGROk to forward your ports..you may not be getting a connection because of your ports are not forwarded. When you are attacking a computer on your network then use 127.0.0.1 or if your on windows call ipconfig an detect your INTERNAL zip address..if you are attacking a machine outside of your network like let's say your girlfriends house then you would use your public option address..your public ip can be found on ip-score.net.

Thanks Philip, as at now i have a cloud server running on rdp with opened personal port, rat wont create connection on targets pc.... i don't think firewall is any issue, already had that checked. no anti virus running on pc as well. I'm running a cafe where i'm the admin and was hoping i could use this RAT to remote my computers. already had AD, DC, ETC... all those suff figured out. RAT's just there.
